doc/manual/import.rst

```diff
@@ -1,17 +1,56 @@
-Adding contest data programmatically
-====================================
+Adding contest data in bulk
+===========================
 
-DOMjudge offers API endpoints to add or update contest data programmatically.
+DOMjudge offers two ways to add or update contest data in bulk: using API
+endpoints and using the jury interface.
 In general, we follow the `CCS specification`_ for all file formats involved.
 
-All of the following examples require you to set up admin credentials in your
-`.netrc`_ file. You need to install `httpie`_ and replace the
+For using the API, the following examples require you to set up admin credentials
+in your `.netrc`_ file. You need to install `httpie`_ and replace the
 ``<API_URL>`` in the examples below with the API URL of your local DOMjudge
 installation.
 
 Importing team categories
 -------------------------
 
+There are two formats to import team categories: a JSON format and a legacy TSV format.
+
+Using JSON
+^^^^^^^^^^
+
+Prepare a file called ``groups.json`` which contains the team categories.
+It should be a JSON array with objects, each object should contain the following
+fields:
+
+- ``id``: the category ID
+- ``name``: the name of the team category
+- ``hidden`` (defaults to ``false``): whether the team category is hidden
+- ``sortorder`` (defaults to ``0``): the sort order of the team category
+
+Example ``groups.json``::
+
+  [{
+    "id": "13337",
+    "name": "Companies",
+    "hidden": true
+  }, {
+    "id": "47",
+    "name": "Netherlands"
```

Comment:
Maybe not too clear an example because country information is also in affiliations...

Reply:
A better idea? WF uses them for super regions but otherwise it is mostly not used, although NWERC indeed uses countries I think (to have a “best from NL”)

Reply:
At least in the past we used categories like "Participants", "Spectators", "Companies" and "Contest staff" (hidden true)